recipe/meta.yaml

```diff
@@ -23,18 +25,28 @@ requirements:
     - pip
   run:
     - python
+    - click
+    - pydantic-settings >=2.3
     - readchar
     - rich
     - typer
 
 test:
   imports:
     - anaconda_cli_base
-  commands:
-    - python -c "from anaconda_cli_base import __version__; assert __version__ == \"{{ version }}\""
-    - pip check
+  source_files:
+    - tests
   requires:
     - pip
+    - pytest
+    - pytest-mock
+    - types-requests
+  commands:
+    - pip check
+    - anaconda -h
+    - anaconda -V
+    - python -c "from anaconda_cli_base import __version__; assert __version__ == \"{{ version }}\""
```

Comment:
because there is `setuptools-scm` do you think it would be good to add the check for `pip` also?
```
python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
```

Reply:
Done